android_sample/build.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+if [[ $ANDROID_SDK_HOME == "" ]]; then
+  echo "Error: set ANDROID_SDK_HOME environment variable and run it again e.g.,"
```

Comment:
Error: ANDROID_SDK_HOME missing, please set env variable.

And error below for NDK